Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabbedMultiView.css

```
@@ -1,98 +1,99 @@
 /* Style applied to the TabbedMultiView. */
 
+:root
+{
+  --tabbedmultiview-topcontrols-padding: 0;
+  --tabbedmultiview-tabstrip-padding: unset;
```

this should be the same padding as view and bottom controls.
```suggestion
  --tabbedmultiview-tabstrip-padding: 0 var(--remotion-themed-spacing-large);
```